internal/controller/dataset_controller.go

```diff
@@ -176,10 +194,9 @@ func (r *DatasetReconciler) loadJob(ctx context.Context, dataset *apiv1.Dataset)
 	}
 
 	if err := r.Cloud.MountBucket(&job.Spec.Template.ObjectMeta, &job.Spec.Template.Spec, dataset, cloud.MountBucketConfig{
-		Name: "dataset",
+		Name: "artifacts",
 		Mounts: []cloud.BucketMount{
-			{BucketSubdir: "data", ContentSubdir: "data"},
-			{BucketSubdir: "logs", ContentSubdir: "logs"},
+			{BucketSubdir: "artifacts", ContentSubdir: "artifacts"},
```

Comment:
Do we still need a `BucketSubdir` now that it is all artifacts together? I think probably cleaner to remove it

Reply:
I dont think we need it anymore

Reply:
Will leave for later refactor